dataset/dictionary/characters.ts

```diff
@@ -157,6 +157,27 @@ export default [
     notes: "イスタロトの称号",
     notesZh: "伊斯塔露的称号",
   },
+  {
+    en: "Naberius",
+    ja: "ナベリウス",  // notes: 魔人任務「背理」で名前が出て来る
```

Comment:
English is preferrable for the comments so that non-Japanese speaking contributors can understand easily.

```suggestion
    ja: "ナベリウス",  // Mentioned in an archon quest "Paralogism"
```